### DISCUSSION TITLE
【求助】鱼要是会抽烟 ，会是怎么抽的，求各位绳匠解答

### DISCUSSION THREAD

**Discussion by halin1117** [share121/inter-knot#2342] [其他]
![IMG_20240816_003400](https://github.com/user-attachments/assets/1199125a-dcf7-4fc1-93ee-8f18bc2be5db)

**Comment by shenguojia123** [comment]
水里面点不了火<img src="https://bbs-static.miyoushe.com/static/2023/11/16/810dd11a4d16759f16ca078596aab6b1_422007669816866994.png" alt="哲-让我想想" width="75" >

**Comment by Xmug** [comment]
插鼻孔里抽

**Comment by March-777th** [comment]
# 如图<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
<img width="465" alt="Snipaste_2024-08-16_18-03-57" src="https://github.com/user-attachments/assets/b6acffe6-a135-4b98-8adb-e2b8baac0233">
<img width="185" alt="Snipaste_2024-08-16_18-04-28" src="https://github.com/user-attachments/assets/f4817bf4-c12d-43e3-8287-280a3f39cff6">

**Comment by Genshin-Neuvillette** [comment]
鱼没有肺怎么抽？
